src/main/java/frc/team2412/robot/commands/shooter/ShooterHardwareTestCommand.java

```diff
@@ -9,7 +9,8 @@
     public ShooterHardwareTestCommand(ShooterSubsystem shooter) {
         addCommands(new ShooterTurretSetAngleCommand(shooter, ShooterConstants.MIN_TURRET_ANGLE),
                 new ShooterTurretSetAngleCommand(shooter, ShooterConstants.MAX_TURRET_ANGLE),
-                new ShooterTurretSetAngleCommand(shooter, 0),
+                new ShooterTurretSetAngleCommand(shooter,
+                        (ShooterConstants.MIN_TURRET_ANGLE + ShooterConstants.MAX_TURRET_ANGLE) / 2),
```

Comment:
If the turret can rotate past 360 degrees, make sure that `MAX_TURRET_ANGLE` is set to > 360 (e.g., 365 degrees instead of 5 degrees) so this command doesn't cause the turret to rotate to an angle it can't reach.

For example, if `MIN_TURRET_ANGLE` is 10 degrees, and the turret can rotate 5 degrees past 360, `MAX_TURRET_ANGLE` should be set to 365 degrees, not 5. Otherwise the turret would try to rotate to 7.5 degrees and possibly break.

This is just a note for the future. The current values of `MIN_TURRET_ANGLE` and `MAX_TURRET_ANGLE` will not trigger this edge case. It might be useful to perform a check for this case upon initialization, but it's not strictly necessary.